pyvrp/cpp/educate/CircleSector.h

```diff
@@ -0,0 +1,82 @@
+#ifndef CIRCLESECTOR_H
```

Comment:
We really cannot avoid this thing, can we? 😢

Reply:
Nope, because it's currently the best thing we have to identify two neighbouring route pairs. We could at some point think of a "RouteNeighbours" object. That could generalize the circle sectors thing, but also implement other route neighbor selection schemes such as % overlapping nodes or some other heuristic.